src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -121,14 +121,11 @@ function rrule(::typeof(/), A::AbstractVecOrMat{<:Real}, B::AbstractVecOrMat{<:R
     C, dC_pb = rrule(adjoint, Cᵀ)
     function slash_pullback(Ȳ)
         # Optimization note: dAᵀ, dBᵀ, dC are calculated no matter which partial you want
-        # this is not a problem if you want the 2nd or 3rd, but if you want the first, it
-        # is fairly wasteful
         _, dC = dC_pb(Ȳ)
-        _, dBᵀ, dAᵀ = dS_pb(extern(dC))
+        _, dBᵀ, dAᵀ = dS_pb(unthunk(dC))
 
-        # need to extern as  dAᵀ, dBᵀ  are generally `Thunk`s, which don't support adjoint
```

Comment:
why did this comment go?

Reply:
It was wrong, we don't exten them

Reply:
we `unthunk` them, but this is now a clear and standard pattern so pointing it out is redundant
things should in general be unthunked before being fed to pullbacks

Reply:
good point! https://github.com/JuliaDiff/ChainRules.jl/issues/177